include/envoy/upstream/cluster_manager.h

```diff
@@ -164,10 +165,10 @@ class ClusterManagerFactory {
   /**
    * Allocate a cluster from configuration JSON.
    */
-  virtual ClusterPtr clusterFromJson(const Json::Object& cluster, ClusterManager& cm,
-                                     const Optional<SdsConfig>& sds_config,
-                                     Outlier::EventLoggerSharedPtr outlier_event_logger,
-                                     bool added_via_api) PURE;
+  virtual ClusterSharedPtr clusterFromJson(const Json::Object& cluster, ClusterManager& cm,
```

Comment:
I've issued an envoy-api PR and will update this PR once that is merged, so that I can get the new envoy-api commit id to use in envoy/bazel/repositories.bzl.